filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest.go

```diff
@@ -11,6 +12,8 @@ import (
 	"github.com/zalando/skipper/filters/openpolicyagent/internal/envoy"
 )
 
+const ResponseHeadersKey = "open-policy-agent:decision-response-headers"
```

Comment:
Please make it unexported (named from lowercase letter `responseHeadersKey`)

Reply:
Thanks for your patience and explanation. Addressed.